spec/mailers/previews/admin_mailer_preview.rb

```diff
@@ -2,4 +2,9 @@ class AdminMailerPreview < ActionMailer::Preview
   def issues_report
     AdminMailer.with(email_address: 'test@blah.com', user: User.admin.last).issues_report
   end
+
+  def missing_reading_window
+    missing = [[AmrDataFeedConfig.where.not(missing_reading_window: nil).first, 10.days]]
```

Comment:
Minor, but after initially running migration all of the configs have a null window so there's an error here